src/test/config/bitbucket/values-EKS.yaml

```diff
@@ -12,13 +12,23 @@ volumes:
     persistentVolumeClaim:
       create: true
   sharedHome:
-    volumeClaimName: efs-claim # Pre-provisioned, and shared by all of our pods
-    customVolume:
-      persistentVolumeClaim:
-        claimName: efs-claim # Pre-provisioned, and shared by all of our pods
-    subPath: ${helm.release.prefix}-bitbucket # Since all of our pods share the same EFS PV, we use subpath mounts to prevent interference
-    nfsPermissionFixer:
-      enabled: true
+    persistentVolume:
+      create: true
+      mountOptions:
+        - rw
+        #    - nfsvers=3 #todo DCNG-946
+        - lookupcache=pos
+        - noatime
+        - intr
+        - rsize=32768
+        - wsize=32768
+        - _netdev
```

Comment:
would it make sense to include any of these values in the PV template?

Reply:
Hmm, I think it does.

Reply:
I've added all of these to the PV template and changed the NFS to v3 in the other PR.